### PR TITLE
Compiler: less agressive inlining

### PR DESCRIPTION
**Pull request by hhugo**
fix https://github.com/ocsigen/js_of_ocaml/issues/1185

### CODE REVIEW

File: compiler/lib/config.ml [hunk 1]
```diff
@@ -118,6 +118,9 @@ module Param = struct
   let switch_max_case =
     p ~name:"switch_size" ~desc:"set the maximum number of case in a switch" (int 60)
 
+  let inlining_limit =
+    p ~name:"inlining-limit" ~desc:"set the size limit for inlining" (int 500)
+
```
Comment (glennsl) [inline compiler/lib/config.ml:123]: What unit is this in? Bytes?

Comment (hhugo) [reply compiler/lib/config.ml:123]: The unit is not really meaningful. What this feature compute is "number of instructions + number of branching"

**Comment by smorimoto** [comment]
FYI: @linoscope

**Comment by linoscope** [comment]
Nice! I will reintroduce inlining and rerun the benchmark once this change is released.  Thanks, @hhugo for working on this!

**Pull request merged**